### DISCUSSION TITLE
Create Seller Account do Develop and Integration??

### DISCUSSION THREAD

**Discussion by DiegoSDeveloper** [amzn/selling-partner-api-models#3612] [General] [answered]
I'm currently developing an integration between an ERP system and SP-API, and I must admit that the process has been quite frustrating. There are numerous steps involved in preparing to start the actual development. Currently, I'm at a stage where I need to create a seller account on Amazon (https://developer-docs.amazon.com/sp-api/docs/registering-your-application). I'm wondering if all of this bureaucracy is necessary, or if I might be taking the wrong approach. I have already created an account as an Amazon developer, but I'm hoping for a simpler option like a sandbox environment where I can perform tests and finalize the integration. This way, the customers of the ERP system could authenticate on Amazon using OAuth and retrieve their list of products and orders.

In Amazon Seller Central, the only option I see is to create a "store," which involves multiple steps such as providing my account details and participating in a video call, among others.

Is this really all there is to integrating with the Amazon marketplace? I'm curious to know if there might be a more streamlined process available.

**Comment by leonardonakagawa** [comment, marked as answer]
Hello @DiegoSDeveloper ,

Thank you for reaching out regarding creating a Seller Account for develop an application with SP-API.

For now, you can only sign up into Seller Central in order to develop applications using SP-API. The process on Seller Central is to thoroughly verify the users signing up and preventing bad-actors.

You can find details regarding this in our documentation on [Registering as a developer](https://developer-docs.amazon.com/sp-api/docs/registering-as-a-developer) and [SP-API Youtube Playlist](https://www.youtube.com/playlist?list=PLyrrqKCT7jFKENJO9n_Y68-5o2GZLgLUU).

If the information available in the documentation link provided above does not fully resolve your inquiry, please [open a support case](https://developer.amazonservices.com/support) with us.

Thanks,
Leonardo Nakagawa
Selling Partner API Developer Support
